tests/test_markup.py

```diff
@@ -13,7 +13,7 @@
 import pickle
```

Comment:
Review is **NOT** needed.
- Converting yield testing into pytest.mark.parametrize
- Removing global